.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  audit_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.2.0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - name: Audit the installed modules
+        run: npm audit --production
```

Comment:
since we use a third-party tool for package auditing in Panther, I don't think we need that anymore